metadata/metadata.go

```diff
@@ -0,0 +1,76 @@
+package metadata
+
+import (
+	"github.com/checkout/checkout-sdk-go/common"
+)
+
+const (
+	metadata = "metadata"
+	card     = "card"
+)
+
+type Format string
+
+const (
+	Basic       Format = "basic"
+	CardPayouts Format = "card_payouts"
+)
+
+type SchemeLocalType string
+
+const (
+	CartesBancaires SchemeLocalType = "cartes_bancaires"
+	Mada            SchemeLocalType = "mada"
+	Omannet         SchemeLocalType = "Omannet"
+)
+
+type PayoutsTransactionsType string
+
+const (
+	NotSupported PayoutsTransactionsType = "not_supported"
+	Standard     PayoutsTransactionsType = "standard"
+	FastFounds   PayoutsTransactionsType = "fast_founds"
+	Unknown      PayoutsTransactionsType = "unknown"
+)
+
+type SourceType string
+
+const (
+	Bin   SourceType = "bin"
+	Card  SourceType = "card"
+	Id    SourceType = "id"
+	Token SourceType = "token"
+)
```

Comment:
This should be relocated in `sources.go` as that's where we use them

Reply:
👍